media/js/mozorg/contribute/friends.js

```diff
@@ -1,4 +1,13 @@
-$(function() {
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, Mozilla) {
+    'use strict';
+
+    var $ffShowSignupForm = $('#ff-show-signup-form');
+    var $ffSignupArea = $('#ff-signup');
+    //var $ffSignupButton = $('#footer_email_submit');
```

Comment:
This is only needed if the commented out GA call below is needed. However, since it would potentially only be used once, will remove and change the commented out GA below to just use `$('#footer_email_submit')` instead of a variable.